client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -722,6 +728,15 @@
                     }
                 }, true);
 
+                $scope.$watch('item.marked_for_not_publication', function(newValue, oldValue) {
+                    if (newValue !== oldValue) {
```

Comment:
Actually a watch _can_ be triggered in the initialization phase with both old and new values equal. If triggering the handler is not desired in such case, this type of check is necessary.